_shows/06_07/zoo_story.md

```diff
@@ -38,7 +38,7 @@ cast:
   - role: Jerry
     name: Magnus McCullagh
   - role: Peter
-    name: Theo Taptiklis
```

Comment:
As he hasn't got much I think we stick with James, may be an idea to stick a comment in here.